### PR TITLE
Corrected description of default SwiftLint behavior in project README

### PR DESCRIPTION
**Pull request by a8c-sab-trial**
*Fixes* an issue identified during discussion of #9129.

*To test:* Read documentation and confirm that Markdown renders as expected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Trying to build the project by itself (WordPress.xcproj) after launching will re
 
 We use [SwiftLint](https://github.com/realm/SwiftLint) to enforce a common style for Swift code. Xcode will show a warning if you don't have it installed. The app should build and work without it, but if you plan to write code, you are encouraged to install it. No commit should have lint warnings or errors.
```
[the remark this answers is not in the document]
Comment (SergioEstevao) [reply README.md]: You can manual run the scheme SwifLint inside Xcode and see the errors. 

Another option that I recommend it's to install the git pre commit hooks, those also run Swift lint before you push and don't allow you to push to the server if there are lint warnings.

[the remark this answers is not in the document]
Comment (SergioEstevao) [reply README.md]: I think it's a good step, @koke do you see any reason why not?

**Comment by koke** [comment]
Oh, we have a rake task to install a pre-commit hook with swiftling already: `rake git:install_hooks`

**Comment by frosty** [comment]
Sorry @koke, I missed that comment. I can amend the readme to mention our rake comment.

**Pull request merged**